infinite_tracing/test/support/record_span_handler.rb

```diff
@@ -43,7 +43,6 @@ def start_handler
             @record_status_stream.push(record_status)
             @record_status_stream.push(nil)
           rescue StandardError => e
-            puts "SERVER ERROR", e.inspect
             raise e
```

Comment:
I just deleted the puts statement because I searched the repo for 'puts', but now that I'm actually looking at this file, this is actually something that was used just by the fake trace observer grpc server, which we aren't using anymore, so I can actually delete the whole file! I'm glad you pointed it out, I didn't realize that was the only thing using it lol.